session.go

```diff
@@ -73,6 +73,24 @@ const (
 	Monotonic Mode = 1
 	// Strong mode is specific to mgo, and is same as Primary.
 	Strong Mode = 2
+
+	// DefaultConnectionPoolSize defines the default maximum number of
+	// connections in the connection pool.
+	//
+	// To override this value set DialInfo.PoolLimit.
+	DefaultConnectionPoolSize = 4096
```

Comment:
Not perfect either but DefaultConnectionPoolLimit is easier to relate back to PoolLimit.